charts/headlamp/values.yaml

```diff
@@ -29,6 +29,7 @@ config:
   # -- base url path at which headlamp should run
   baseURL: ""
   oidc:
+    create: false
```

Comment:
I think we can achieve the same output without having to introduce a new variable here. We can use the `with` helm template operator to achieve this.
https://helm.sh/docs/chart_template_guide/variables/

Reply:
Sure, updated